chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/rules/developer/DeveloperEditTransparencyAttestationValidation.java

```diff
@@ -47,17 +48,24 @@ private boolean isTransparencyAttestationUpdated(final DeveloperDTO original, fi
             return true;
         } else {
             for (DeveloperACBMapDTO originalMapping : original.getTransparencyAttestationMappings()) {
-                Optional<DeveloperACBMapDTO> matchingMapping =
-                        changed.getTransparencyAttestationMappings().stream()
-                    .filter(changedMapping -> StringUtils.equals(originalMapping.getAcbName(), changedMapping.getAcbName()))
-                    .findFirst();
+                Optional<DeveloperACBMapDTO> matchingMapping = changed.getTransparencyAttestationMappings().stream()
+                        .filter(changedMapping -> StringUtils.equals(originalMapping.getAcbName(), changedMapping.getAcbName()))
+                        .findFirst();
                 if (!matchingMapping.isPresent()
-                        || !StringUtils.equals(originalMapping.getTransparencyAttestation(),
+                        || !areTransparencyAttestationsEqual(originalMapping.getTransparencyAttestation(),
                                 matchingMapping.get().getTransparencyAttestation())) {
                     return true;
                 }
             }
         }
         return false;
     }
+
+    private Boolean areTransparencyAttestationsEqual(TransparencyAttestationDTO a, TransparencyAttestationDTO b) {
+        if (a == null || b == null) {
```

Comment:
Shouldn't this be `&&`? How can they be equal if one exists and the other doesn't?